src/ejabberd_c2s.erl

```diff
@@ -1246,7 +1246,7 @@ handle_info({store_session_info, User, Server, Resource, KV, _FromPid}, StateNam
 handle_info(Info, StateName, StateData) ->
     handle_incoming_message(Info, StateName, StateData).
 
-%%% incoming messages
+%%% incoming messages from other users or services to this device
```

Comment:
Me Gusta! At last a clear definition in at least one place what do "incoming" and "outgoing" mean. :D

Reply:
device ?